source/Ship.cpp

```diff
@@ -2804,7 +2805,7 @@ bool Ship::IsReadyToJump(bool waitingIsReady) const
 		return false;
```

Comment:
I can't comment high enough in the preview, but there's a line just above here that calls JumpFuel, which just ends up calling GetCheapestJumpType, so this code should look something like this:
```c++
pair<Ship::JumpType, double> jumpUsed = GetCheapestJumpType(targetSystem);
double fuelCost = jumpUsed.second;
if(!fuelCost || fuel < fuelCost)
	return false;

Point direction = targetSystem->Position() - currentSystem->Position();
bool isJump = (jumpUsed.first == JumpType::JumpDrive);
...
```

Reply:
done